src/block.rs

```diff
@@ -391,7 +393,7 @@ impl TryConvert for Proc {
 ///
 /// This effectivly makes the closure's lifetime managed by Ruby. It will be
 /// dropped when the returned `Value` is garbage collected.
-fn wrap_closure<F, R>(func: F) -> (*mut F, Value)
+pub(crate) fn wrap_closure<F, R>(func: F) -> (*mut F, Value)
```

Comment:
Will want to re-use this functionality for `Fiber::new` later. Happy to un-pub it in this PR if you want to, though.